app/Livewire/Project/New/CoolifyJsonImport.php

```
@@ -0,0 +1,154 @@
+
+            return;
+        }
+
+        // Get destination
+        $destination_uuid = $this->query['destination'];
+        $destination = StandaloneDocker::where('uuid', $destination_uuid)->first();
+        if (! $destination) {
+            $destination = SwarmDocker::where('uuid', $destination_uuid)->first();
+        }
+        if (! $destination) {
+            throw new \Exception('Destination not found. What?!');
```

PHPMD (2.15.0)

Missing class import via use statement (line '98', column '23'). (undefined)

(MissingImport)